### DISCUSSION TITLE
The  O̶f̶f̶i̶c̶i̶a̶l̶ List Of Rejected Suggestions

### DISCUSSION THREAD

**Discussion by GrahamSH-LLK** [ScratchAddons/ScratchAddons#927] [Discussion]
# The Official List Of Rejected Suggestions

## 1. Themes
1. Custom themes and or pink themes

## 2. Options page
1. Ability to load addons from files (#287 )
There are technical restrictions that forbid execution of dynamic code.
2. Custom icons for addons (#170)
Setting pages are not extension "stores".

## 3. Addons
1. View deleted forum post (#111)
2. Character replacer for typing more in comments (#187)
It would break screen readers and may trigger bad word filters unintentionally.
3. Allow cloud variable modification after seeing inside (#217)
This allows cheating, which is against rules.
4. Profile picture renderer (#698)
This uses external servers, which is hard to manage.
5. Passing variable references to custom blocks (#697)
The amount of changes and monkey-patches requires are too big.
6. Project templates (#717)
Remixing and copying exists.
7. Private studios (#696)
There is no method to achieve private studios using Scratch functionality.
8. Adding new block (#672)
Blocks that do not exist in vanilla Scratch would not execute without Scratch Addons. Also, this is against Terms of Use. Adding existing but hidden blocks (such as while loop) or adding a block as a custom block is not rejected.
9. Markdown in comments (#184)
It is useless since not everyone has Scratch Addons.
10. **A button to invite all followers to a studio** (#635)
"There are rate limits, and I don't think any type of automatic inviting would be ok."
11. Delete all projects from a studio
Rate limits would make this useless.

## 4. General
1. Safari support (#340)
We do not have money to purchase a developer account, and Safari lacks some features.

**Comment by WorldLanguages** [comment]
Objection. I ~did not~ approve this. Hence it's ~not~ official. Hehe.

**Comment by Samq64** [comment]
Is this official or not?

**Reply by GrahamSH-LLK** [reply]
Basically. It's not fully up to date though.
